birdhouse/config/magpie/providers.cfg.template

```diff
@@ -62,6 +62,26 @@ providers:
     c4i: false
     type: thredds
     sync_type: thredds
+    configuration:
+      skip_prefix: "thredds"  # prefix to ignore, below prefixes will be matched against whatever comes after in path
+      file_patterns:
+        # note: make sure to employ quotes and double escapes to avoid parsing YAML error
+        - ".+\\.nc"
+      metadata_type:
+        prefixes:
+          - null  # note: special YAML value evaluated as `no-prefix`, use quotes if literal value is needed
+          - "\\w+\\.gif"  # threddsIcon, folder icon, etc.
+          - "\\w+\\.ico"  # favicon
+          - "\\w+\\.txt"  # licence
+          - "\\w+\\.css"  # tds.css
+          - "catalog\\.\\w+"  # note: special case for `THREDDS` top-level directory (root) accessed for `BROWSE`
+          - catalog
+          - ncml
```

Comment:
I have a vague  memory there are more here?

Reply:
Could be, those are the ones I was asked to put via other comments in original PR that added that feature.

Reply:
The list was right here https://github.com/Ouranosinc/Magpie/pull/353#issuecomment-710495259.  Missing uddc and iso.